src/services/codefixes/importFixes.ts

```diff
@@ -286,8 +286,11 @@ function createImportAdderWorker(sourceFile: SourceFile | FutureSourceFile, prog
         const checker = program.getTypeChecker();
         const symbol = checker.getMergedSymbol(skipAlias(exportedSymbol, checker));
         const exportInfo = getAllExportInfoForSymbol(sourceFile, symbol, symbolName, moduleSymbol, /*preferCapitalized*/ false, program, host, preferences, cancellationToken);
+
+        // If no exportInfo is found, this means the export could not be resolved, so we should not generate an import
+        if (!exportInfo) return;

```

Comment:
Maybe we should at least ensure this never happens unless there are `preferences.autoImportFileExcludePatterns`